### ISSUE TITLE
Fails to open files with a space in its name.

### ISSUE THREAD

**Issue by orbea** [lecram/rover#22]
OS: `Slackware64-current`
`rover-0.4.2-x86_64-1_SBo`

When trying to open a file with rover using a program like less it will fail to do so if the file name has spaces in it.

To test try this, `touch "some test name"` and then opening it in rover and you will see an issue along the lines of:
```
$ rover
some: No such file or directory
test: No such file or directory
name: No such file or directory
```

**Comment by lecram** [comment]
Thanks for reporting this. The bug was introduced when Rover started launching external programs through a shell by default (version 0.4.1). It should be fixed now (see commits 486a1459319a656b0b4b218c6451e629858d6060 and 21152dd558248363f497649e01611fccbb89f974).

**Comment by orbea** [comment]
Thank you for the quick fix, I have tried the two new commits and can confirm they fix the problem.
